files/default/solr-logrotate

```diff
@@ -0,0 +1,10 @@
+/var/log/solr/*log {
+    daily
+    rotate 10
+    missingok
+    notifempty
+    compress
+    postrotate
+        service solr restart
```

Comment:
why does it need solr restart? is it not rotating with log4j?

Reply:
Some logs are log4j, some aren't.